src/api/app/views/webui/package/_files_view.html.haml

```diff
@@ -23,6 +23,7 @@
       = link_to(new_project_package_files_path(project, package)) do
         %i.fas.fa-plus-circle.text-primary
         Add File
+    = sprintf("checkout this package with osc -A %s checkout %s/%s && cd $_", "FIXME: https://api.opensuse.org", project, package)
```

Comment:
= "Checkout this package with osc -A #{api_url} checkout #{project}/#{package} && cd $_"